test/framework/src/main/java/org/elasticsearch/test/SerializationTestUtils.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.test;
+
+import org.elasticsearch.Version;
+import org.elasticsearch.common.io.stream.InputStreamStreamInput;
+import org.elasticsearch.common.io.stream.OutputStreamStreamOutput;
+import org.elasticsearch.common.io.stream.Writeable;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+/**
+ * Utility methods for assisting with testing {@link Writeable} object.
+ */
+public class SerializationTestUtils {
```

Comment:
This is a drive-by comment. I can't find any uses of this? And also, I think that `AbstractWireSerializingTestCase` provides roughly the same functionality?

Reply:
It's used in 
- idp.action.DeleteSamlServiceProviderRequestTests
- idp.action.PutSamlServiceProviderRequestTests
- idp.saml.sp.SamlServiceProviderDocumentTests
- idp.saml.support.SamlAuthenticationStateTests

You're right, I can switch those to use the existing methods.
I was sure we had something like that, but I just couldn't find it, so I figured I must have remembered incorrectly.

Reply:
Strange, the IDE showed me there were not any uses. Either way, thanks for switching over.